src/opower/utilities/enmax.py

```diff
@@ -0,0 +1,85 @@
+"""Enmax"""
```

Comment:
nit: Add an empty line after this and add missing period.

Run:
`isort . ; black . ; flake8 . ; ruff . --fix ; mypy --install-types .`